abel/onion_peeling.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+from scipy.ndimage.interpolation import shift
+
+################################################################################
+#
+# Onion peeling algorithm, also known as "back projection"
+#
+# This algorithm was adapted by Dan Hickstein from the original Matlab 
+# implementation, created by Chris Rallis and Eric Wells of 
+# Augustana University, and described in this paper:
+#
+#  http://scitation.aip.org/content/aip/journal/rsi/85/11/10.1063/1.4899267
+#
+# The algorithm actually originates from this 1996 RSI paper by Bordas et al:
+#
+#  http://scitation.aip.org/content/aip/journal/rsi/67/6/10.1063/1.1147044
+#
+# 2016-03-19: Jacobian intensity correction, allow negative image values
+#             Steve Gibson and Dan Hickstein
+# 2015-12-16: Python version of the Matlab code, by Dan Hickstein see issue #56
+#
+################################################################################
+
+def init_abel(xc, yc):
```

Comment:
should this be a private member to keep it out of the docs? Just put a underscore before the name.